### DISCUSSION TITLE
FR: Add an option to give the menu a hover dropdown.

### DISCUSSION THREAD

**Discussion by mikesnoeren** [craftcms/cms#8727] [Ideas]
When for example clearing cache you have to perform too many actions: 
Click utilities > click clear cache in the submenu > click clear caches.

Perhaps this could be simplified by using an optional dropdown menu on hover(or click) so it doesn't require extra page loads. Especially when using an external database while developing. this will speed up the process of navigation a lot!

**Comment by AugustMiller** [comment]
FYI, you can clear caches via the CLI!

```
$ ./craft clear-caches/all
```

**Comment by davidhellmann** [comment]
@Mikesnoeren you need this: https://plugins.craftcms.com/cp-clearcache
